sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -231,4 +233,28 @@ export class ApplicationAESTController extends BaseController {
       applicationId,
     );
   }
+
+  /**
+   * Get history of application versions for an application
+   * where the current application is excluded.
+   * @param applicationId application Id.
+   * @returns history of application versions.
+   */
+  @ApiNotFoundResponse({
+    description: `Application not found or not on ${ApplicationStatus.Completed} status.`,
+  })
+  @Get(":applicationId/versions")
+  async getApplicationVersionHistory(
+    @Param("applicationId", ParseIntPipe) applicationId: number,
+  ): Promise<ApplicationVersionAPIOutDTO[]> {
+    const applications =
+      await this.applicationService.applicationVersionHistory(applicationId);
+    if (!application) {
```

Comment:
This is the application from express. 

![image](https://github.com/user-attachments/assets/a55431ee-465e-41b9-8f5c-4065f8758faf)